doc/changelog.md

```diff
@@ -214,101 +310,37 @@
+    some definitions persisted that may be different than ones used by third party implementors. To
+    prevent symbol conflicts QIR runner's stdlib was removed and is no longer linked against. As a
+    result, the following functions are now defined and implemented in Catalyst's runtime:
 
-  QIR encodes quantum instructions as LLVM function calls. This allows frontends to generate
-  QIR, middle-ends to optimizie QIR, and code generators to lower these function calls into
-  appropriate instructions for the target. One of the possible implementations for QIR is to
-  lower QIR instructions to function calls with the same signature and implement QIR as a library.
-  Other implementations might choose to lower QIR to platform specific APIs, or replace function
-  calls with semantically equivalent code.
-
-  Catalyst implemented QIR as a library that can be linked against a QIR module.
-  This works great when Catalyst is the only implementor of QIR.
-  However, when other QIR implementors, who also lower implement a quantum runtime as functions to be
-  linked against, this may generate symbol conflicts.
-
-  This PR changes the runtime such that QIR instructions are now lowered to functions where
-  the `__quantum__` part of the function name is replaced with `__catalyst__`. This prevents
-  the possibility of symbol conflicts with other libraries that implement QIR as a library.
-  However, it doesn't solve it completely. Since the `__catalyst__` functions are still exported.
-  If another library implemented the same symbols exported by the runtime, the same problem would
-  presist.
-
-* The Catalyst runtime no longer depends on QIR runner's stdlib.
-  [(#470)](https://github.com/PennyLaneAI/catalyst/pull/470)
+    * `int64_t __catalyst__rt__array_get_size_1d(QirArray *)`
+    * `int8_t *__catalyst__rt__array_get_element_ptr_1d(QirArray *, int64_t)`
 
-  Similar to changing the runtime API for QIR instructions, we no longer depend nor link against
-  QIR runner's stdlib. With PR #464, most of the symbol conflicts were resolved, but by linking
-  against QIR runner's stdlib, some definitions persisted that may be different than ones
-  used by third party implementors. To prevent symbol conflicts QIR runner's stdlib was removed
-  and is no longer linked against. As a result, the following functions are now defined and
-  implemented in Catalyst's runtime:
-  * `int64_t __catalyst__rt__array_get_size_1d(QirArray *)`
-  * `int8_t *__catalyst__rt__array_get_element_ptr_1d(QirArray *, int64_t)`
+    and the following functions were removed since the frontend does not generate them
 
-  and the following functions were removed since the frontend does not generate them
-  * `QirString *__catalyst__rt__qubit_to_string(QUBIT *)`
-  * `QirString *__catalyst__rt__result_to_string(RESULT *)`
+    * `QirString *__catalyst__rt__qubit_to_string(QUBIT *)`
+    * `QirString *__catalyst__rt__result_to_string(RESULT *)`
 
 <h3>Bug fixes</h3>
 
```

Comment:
@erick-xanadu I can't comment on it, but what does the changelog item directly below (#555) mean?

Reply:
Do you mean this?

>  Applying `map_wires` to `QCtrl` lead to an incorrect circuit. The wires in the nested regions remained unchanged.

What this means is that the implementation of `map_wires` (which `QCtrl` inherited from `Operation`) was wrong. This is because `QCtrl` contains a nested tape and did not apply `map_wires` to any of its nested tapes. All HybridOps should implement this function. I have added this ticket: https://github.com/PennyLaneAI/catalyst/issues/556

Reply:
Does my comment answer your question?

Reply:
Yep! Although I'd be curious if this was a bug a user could access, and if so we could include a short example

Reply:
I think definitely. Let me come up with an example.

Reply:
```python
import pennylane as qml
from pennylane.tape import QuantumTape
from catalyst import qjit, cond
from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
from pennylane.transforms.core import transform

dev = qml.device("lightning.qubit", wires=2)

@transform
def custom_transform(tape: QuantumTape) -> (Sequence[QuantumTape], Callable):
    new_ops = []
    for op in tape:
        new_op = op.map_wires({0: 1, 1: 0})
        new_ops.append(new_op)

    new_tape = QuantumTape(new_ops, tape.measurements)
    return [new_tape], lambda res: res[0]


@qjit
@custom_transform
@qml.qnode(dev)
def qnn(c):
    @cond(c)
    def foo():
        qml.RX(3.14, wires=[1])
    foo()
    return qml.state()

print(qnn(True))
```